final/GitHubSearch/GitHubSearchApp.swift

```diff
@@ -0,0 +1,17 @@
+import SwiftUI
+
+import ComposableArchitecture
+
+@main
+struct GitHubSearchApp: App {
+  var body: some Scene {
+    WindowGroup {
+      RepoSearchView(
+        store: Store(
+          initialState: RepoSearch.State(),
+          reducer: RepoSearch()
```

Comment:
```suggestion
          reducer: RepoSearch()._printChanges()
```

State변경이 콘솔에 나오면 더 좋을 것 같습니다.